docs/reference/content/driver/tutorials/bulk-writes.md

```diff
@@ -25,25 +25,25 @@ operations:
 ```java
 // 1. Ordered bulk operation - order is guaranteed
 collection.bulkWrite(
-  Arrays.asList(new InsertOneModel<>(new Document("_id", 4)),
-                new InsertOneModel<>(new Document("_id", 5)),
-                new InsertOneModel<>(new Document("_id", 6)),
-                new UpdateOneModel<>(new Document("_id", 1),
+  Arrays.asList(new InsertOneModel<Document>(new Document("_id", 4)),
```

Comment:
These were left out intentionally, as since Java 7 the type can be inferred by the compiler.